rules/regexp/headers.lua

```diff
@@ -917,3 +923,36 @@ reconf['HAS_XOIP'] = {
   score = 0.0,
   group = 'headers'
 }
+
+reconf['HAS_LIST_UNSUB'] = {
+  re = string.format('%s', 'header_exists(List-Unsubscribe)'),
+  description = 'Has List-Unsubscribe header',
+  score = -0.01,
+  group = 'headers'
+}
+
+reconf['HAS_GUC_PROXY_URI'] = {
+   re = '/\\.googleusercontent\\.com\\/proxy/{raw_mime}i',
```

Comment:
Why not a URL regex?

Reply:
Honestly - I can't remember.

Reply:
Can we change it?

Reply:
Of course - once the new version is out, then I'll do a corpus run and check it against the previous results.  I doubt it will make any difference to the result.

Reply:
Ok - done.

Reply:
Cool, thank you!